examples/windows/kubernetes-hyperv.json

```diff
@@ -5,33 +5,25 @@
       "orchestratorType": "Kubernetes",
       "orchestratorRelease": "1.15",
       "kubernetesConfig": {
-               "apiServerConfig" : {
-                 "--feature-gates": "HyperVContainer=true"
-               },
-               "kubeletConfig" : {
-                    "--feature-gates": "HyperVContainer=true"
-                }
-          }
+        "networkPlugin": "azure",
+        "containerRuntime": "containerd",
+        "windowsContainerdURL": "https://k8swin.blob.core.windows.net/k8s-windows/containerd/containerplat-aks-test-0.0.8.zip"
```

Comment:
May I regard this package url as the _official url_for 20H1 so far?

Reply:
yes this is the latest package

Reply:
These are the official bits but we will probably need to host them along with our other artifacts for production use.
k8swin.blob.core.windows.net is just hosted in a single zone.

Reply:
I am going to set up a different url for prod use. Will follow back once I have it

Reply:
Sounds good.
That should not block this PR IMO